spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -190,9 +222,24 @@ private Map setupPySparkEnv() throws IOException{
       }
     }
 
+    LOGGER.debug("PYTHONPATH: " + env.get("PYTHONPATH"));
     return env;
   }
 
+  // Run python shell
+  // Choose python in the order of
+  // PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON > zeppelin.pyspark.python
```

Comment:
Zeppelin doesn't use spark-submit to launch python process, instead it launch python via its own logic. That's why I want to use `PYSPARK_DRIVER_PYTHON` first to make it consistent with `spark-submit`. Ideally `zeppelin.pyspark.python` should be removed IMO, but I have to keep it here now for compatibility. IMO zeppelin should not introduce new properties that have  the duplicated property in spark.